modules/rosa-sdpolicy-am-regions-az.adoc

```diff
@@ -22,7 +22,7 @@ endif::rosa-with-hcp[]
 
 [NOTE]
 ====
-China and GovCloud (US) regions are not supported, regardless of their support on OpenShift 4.
```

Comment:
Can you add a note that for GovCloud regions the customer will need to visit this page https://console.redhat.com/openshift/create/rosa/govcloud to request access?